### PR TITLE
Basic html pages

### PR DESCRIPTION
**Pull request by Stokalas**
**Created:**
 - Boiler plate _Layout.cshtml file to keep all files simililarly structured
 - Index page for website with no functionality
 - LogIn page with no functionality

**Edited:**
 - Changed some html attributes to asp.net attributes (mainly href to asp-page)
 - Added Razor.RuntimeComplitaion Nugget package

 **Functionality:**
 - You can switch between Index and LogIn pages through links

**What was used:**
 - HTML5
 - CSS
 - Bootstrap 4
 - Generated C# asp.net code on creation of project.

_Would be nice if everyone would get familiar with code and point out faulty places. 🙂_ 

**MAIN FILES TO LOOK AT:**

-Index.cshtml
-_Layout.cshtml
-logIn.cshtml
-styles.css
_Others can be ignored since it is auto generated_

### CODE REVIEW

File: src/Pages/Index.cshtml [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+            </div>
+            <a class="carousel-control-prev" href="#testimonial-carousel" role="button" data-slide="prev">
+                <span class="carousel-control-prev-icon" aria-hidden="true"></span>
+                <span class="sr-only">Previous</span>
+            </a>
+            <a class="carousel-control-next" href="#testimonial-carousel" role="button" data-slide="next">
+                <span class="carousel-control-next-icon" aria-hidden="true"></span>
+                <span class="sr-only">Next</span>
+            </a>
+        </div>
```
Comment (ciliakas) [inline src/Pages/Index.cshtml:87]: I see there's a end tag missing here for section, is this on purpose?

File: src/Pages/Shared/_Layout.cshtml [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+                        <div class="dropdown-divider"></div>
+                        <a class="dropdown-item" href="#">Goals</a>
+                    </div>
+                </li>
+                <li class="nav-item dropdown">
+                    <a class="nav-link active dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                        Reports
+                    </a>
+                    <div class="dropdown-menu" aria-labelledby="navbarDropdown">
+                        <a class="dropdown-item" href="#">Spendings</a>
```
Comment (ciliakas) [inline src/Pages/Shared/_Layout.cshtml]: Typo in 'spending**s**'

**Review by ciliakas — APPROVED** [review]
So I've spent some time beforehand learning how the structure of this type of project works, so I think I more or less got the basic idea of it.
Since I only have very rudimentary understand of HTML, can't comment on it too much. Only thing I noticed was an end tag missing in Index.cshtml.

File: src/Pages/Index.cshtml (hunk 1 above)
Comment (Stokalas) [reply src/Pages/Index.cshtml:87]: It is probably a mistake left when I moved the "boilerplate" code. Thanks for pointing it out!

**Review by ciliakas — APPROVED** [review]

**Pull request merged**